com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1417,16 +1415,41 @@ internal void HandleApproval(ulong clientId, bool createPlayerObject, ulong? pla
                     OnClientConnectedCallback?.Invoke(clientId);
                 }
 
-                if (!createPlayerObject || (playerPrefabHash == null && NetworkConfig.PlayerPrefabHash == null)) return;
+                //We are done if we don't automatically spawn the player or there is an issue with the configuration (i.e. no player prefab hash
+                if (!createPlayerObject || (playerPrefabHash == null && NetworkConfig.PlayerPrefabHash == null))
+                {
+                    if(createPlayerObject)
+                    {
+                        //Provide meaningful feedback to developers that this happened.
+                        NetworkLog.LogWarning("NetworkManager configured to spawn player prefab but no prefab is defined or there is a problem with the prefab hash.");
```

Comment:
This is interesting.  Is this different behavior than now, where if you don't have your default player spawn it crashes?  Now you get a warning?